### PR TITLE
Upgrade dependencies.

### PR DESCRIPTION
**Pull request by jgerigmeyer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,64 +65,64 @@
...
-    "@babel/preset-env": "^7.14.5",
-    "@testing-library/dom": "^7.31.2",
-    "@testing-library/jest-dom": "^5.14.1",
-    "@yarnpkg/pnpify": "^2.4.0",
+    "@babel/core": "^7.16.7",
+    "@babel/preset-env": "^7.16.7",
+    "@testing-library/dom": "^8.11.1",
+    "@testing-library/jest-dom": "^5.16.1",
+    "@yarnpkg/sdks": "^2.6.0-rc.1",
     "accoutrement": "^3.0.1",
```
Comment (jgerigmeyer) [inline package.json]: @mirisuzanne I think upgrading Accoutrement is the remaining task here? Right now there are a lot of warnings: `Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.`

**Comment by mirisuzanne** [comment]
I made the Accoutrement update, and assigned to @dvdherron and @stacyk for review (probably only needs one or the other). We could wait to merge until Accoutrement is out of beta, or merge and make any final adjustments at that point.

**Review by dvdherron — APPROVED** [review]
looks great to me. 

some very minor comments. 

And one other small thing: it wasn't obvious to me what the change was that caused it, but these status links are now vertically stacked 

bild

instead of sitting side-by-side
bild

**Comment by jgerigmeyer** [comment]
> And one other small thing: it wasn't obvious to me what the change was that caused it, but these status links are now vertically stacked
> 
> bild
> 
> instead of sitting side-by-side bild

Weird! I see the same thing, but the markup and relevant styles look unchanged to me. Any ideas @mirisuzanne?

**Comment by mirisuzanne** [comment]
@jgerigmeyer @dvdherron I bet it's a difference between the old accoutrement-init reset, and css-remedy. If I remember right, css-remedy applies block display to svgs? That may not really be a good default, but should also be easy to override.

**Comment by mirisuzanne** [comment]
@jgerigmeyer @dvdherron I think that addresses the review comments. I also set up a staging deployment for us…

File: package.json [hunk 2]
```diff
@@ -70,7 +70,7 @@
     "@testing-library/dom": "^8.11.1",
     "@testing-library/jest-dom": "^5.16.1",
     "@yarnpkg/sdks": "^2.6.0-rc.2",
-    "accoutrement": "4.0.0-beta.2",
+    "accoutrement": "pre",
```
Comment (jgerigmeyer) [inline package.json:73]: This makes me nervous, since beta releases may contain breaking changes.

**Review by jgerigmeyer — COMMENTED** [review]
@mirisuzanne Comparing https://deploy-preview-368--herman-staging.netlify.app/ and https://www.oddbird.net/herman/docs/ I notice a number of spacing changes (mostly that this PR adds more vertical whitespace, and larger tab size for indentation). But the most notable "broken" changes relate to `@example`/code blocks, and iframes.

Before:

Screen Shot 2022-01-20 at 5 12 05 PM

After (wrapped, can't scroll):

Screen Shot 2022-01-20 at 5 12 09 PM

Screen Shot 2022-01-20 at 5 12 42 PM

Before:

Screen Shot 2022-01-20 at 5 14 04 PM

After (borders):

Screen Shot 2022-01-20 at 5 14 09 PM

**Comment by stacyk** [comment]
> I notice a number of spacing changes (mostly that this PR adds more vertical whitespace, and larger tab size for indentation).

The missing init strikes again?

**Comment by mirisuzanne** [comment]
@stacyk yep. @jgerigmeyer I think I fixed the main issues I could find there, now?

**Review by jgerigmeyer — COMMENTED** [review]
@mirisuzanne Better. There are still some font-size/spacing differences (not a big deal), and compiled examples now wrap instead of side-scroll:

Before (https://www.oddbird.net/herman/docs/api_json-export):

Screen Shot 2022-01-21 at 3 26 07 PM

After (https://deploy-preview-368--herman-staging.netlify.app/api_json-export.html):

Screen Shot 2022-01-21 at 3 25 48 PM

**Comment by dvdherron** [comment]
Outside of the wrapping of the compiled examples, this looks good to me. 

It looks like this line from [CSS Remedy](https://github.com/jensimmons/cssremedy/blob/468e31a7eda599eea2003ed7983c190828ffb5fd/css/remedy.css#L96) needs to be overriden:

`pre { white-space: pre-wrap; }`

**Comment by mirisuzanne** [comment]
@dvdherron I think you're right… But I also wonder if wrapping is better than overflow? I guess there's just tradeoffs. Do you (or @stacyk/@SondraE) have a preference for example code overflowing vs wrapping?

**Comment by jgerigmeyer** [comment]
> @dvdherron I think you're right… But I also wonder if wrapping is better than overflow? I guess there's just tradeoffs. Do you (or @stacyk/@SondraE) have a preference for example code overflowing vs wrapping?

@mirisuzanne Formatting (line breaks, tabs, etc) matters in code -- I think it's important to overflow rather than wrap.

**Comment by mirisuzanne** [comment]
@jgerigmeyer fixed.

**Review by jgerigmeyer — COMMENTED** [review]
Looks good! I have a slight preference for less vertical whitespace, but it's not a big deal.

Before:

Screen Shot 2022-01-24 at 4 46 56 PM

After:

Screen Shot 2022-01-24 at 4 46 50 PM

**Comment by mirisuzanne** [comment]
@jgerigmeyer, that's another result of changing resets. It's an easy fix. Are there are any other changes that stand out to you, before I go back in? 😅

**Comment by jgerigmeyer** [comment]
> @jgerigmeyer, that's another result of changing resets. It's an easy fix. Are there are any other changes that stand out to you, before I go back in? 😅

@mirisuzanne Not that I can see! I'll note that I did mention the spacing differences in both of my [other](https://github.com/oddbird/sassdoc-theme-herman/pull/368#pullrequestreview-858863880) [reviews](https://github.com/oddbird/sassdoc-theme-herman/pull/368#pullrequestreview-859996924) already, so it's not like I'm intentionally slow-playing it. 😄

**Comment by mirisuzanne** [comment]
@jgerigmeyer haha, fair. Ok, see how that looks.

Also, in the future, any way we can see these diffs without 2000+ files changed?

**Review by jgerigmeyer — COMMENTED** [review]
LGTM!

> Also, in the future, any way we can see these diffs without 2000+ files changed?

Hm, not very easily if we group dependency upgrades. 🤔

**Pull request merged**